scripting/shavit-zones.sp

```diff
@@ -479,7 +487,10 @@ public void UnloadZones(int zone)
 			gV_MapZones[zone][1][i] = 0.0;
 		}
 	}
-
+	else if(zone == view_as<int>(Zone_CustomSpawn))
+	{
+		ClearCustomSpawn();
+	}
 	else
```

Comment:
Please make the spacing identical to the rest of the code (one new line before both `else`s)